.github/runners/Dockerfile.ci

```diff
@@ -62,13 +63,26 @@ RUN apt-get update && apt-get install -y \
 RUN ln -sf /bin/bash /bin/sh
 
 # Build and install the most recent version of the mold linker
-RUN https_proxy=${HTTPS_PROXY} git clone https://github.com/rui314/mold.git /tmp/mold
-RUN cd /tmp/mold && \
-    git checkout $MOLD_VERSION && \
-    mkdir build && \
-    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=c++ -B build && \
-    cmake --build build -j$(nproc) && \
-    sudo cmake --build build --target install
+RUN https_proxy=${HTTPS_PROXY} curl -L -O --output-dir /tmp https://github.com/rui314/mold/releases/download/v$MOLD_VERSION/mold-$MOLD_VERSION-$(arch)-linux.tar.gz
+RUN cd /tmp && \
+    sudo tar -C /usr/local --strip-components=1 -xzf mold-$MOLD_VERSION-$(arch)-linux.tar.gz
```

Comment:
downloading pre-built binaries instead of compiling to save time during devcontainer build